modal/image.py

```diff
@@ -1396,7 +1396,7 @@ def build_dockerfile(version: ImageBuilderVersion) -> DockerfileSpec:
 
     def run_function(
         self,
-        raw_f: Callable[..., Any],
+        raw_f: Callable[..., None],
```

Comment:
This is a narrowing but I can't think of a case where returning something is desired. These should just be side-effect functions.